.github/workflows/nightly-merge.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Nightly Merge (master to iiot)'
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
this is UTC, so it would be 5p, PDT. Just want to make sure that we are ok with that.

Reply:
Another question - can it be done on every merge to master? I might reduce the possibility of conflicts. Does it make sense? //cc @dmolokanov @varunpuranik

Reply:
I think a daily merge is good enough.  @varunpuranik please confirm with time you prefer the auto-merge to run.  thanks.

Reply:
I think we will not diverge too much within a day. But we don't want to hoard all resources to run CI and e2e tests for both branches `iiot` and `master` at the same rate. Nighty should reduce load on our infrastructure.

As a side note, it'd be really nice that this merges follow PR flow with auto-completion. Probably, it'll give more visibility especially if it is a conflict to be resolved.

Scenario be like: 
1. bot creates a pr for `master` to `iiot` changes with `ready-to-merge` label
2. github spins up all checks and if all checks are successful
3.  Kodiak merges changes **without squash**

Reply:
Will commit and try it out first. We can change the time later.